styles/index.css

```diff
@@ -160,6 +160,13 @@ body {
     font-size: 16px;
 }
 
+@media (max-height: 800px) and (min-width: 1024px) {
+    html {
+        font-size: 12px;
```

Comment:
I saw some comments from the trading comp about the text being too small, was it related to this?

It also shifts the positioning of things quite a bit.